src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -59,6 +59,23 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->cannotBeEmpty()
                     ->scalarPrototype()
                         ->end()
+                    ->validate()
+                        ->ifTrue(static function (array $paths): bool {
+                            $filename = null;
+                            foreach ($paths as $path) {
+                                $currentFilename = basename($path, '.scss');
+
+                                if ($filename === $currentFilename) {
+                                    return true;
+                                }
+
+                                $filename = $currentFilename;
```

Comment:
Hm, I'm not sure this solution is robust enough... it works for 2 files, but with more files it will not work because it only reveals duplicates that are going consequently.

I think we need to collect all the filenames in an array, and every time check if that array already contains the currently handled filename. It should be more robust, but still it might be a possible problem later if the upstream logic changed :/

Reply:
Thanks for (again) being sharper than me, fixed it and even added tests for the configuration